lib/dk/local.rb

```diff
@@ -5,6 +5,8 @@ module Dk::Local
 
   class BaseCmd
 
+    attr_reader :scmd
+
```

Comment:
@kellyredding your explanation makes sense and like you say it's probably nice to have it for edge cases. I would've been ok with exposing the `start` and `wait` but it does bring a lot of extra "noise" like, should those be added to the task so I can start commands? Anyway, I don't think it's an issue to expose this.